package.json

```diff
@@ -44,7 +44,7 @@
     "css-loader": "^0.28.4",
     "detect-passive-events": "^1.0.2",
     "dotenv": "^4.0.0",
-    "emoji-mart": "Gargron/emoji-mart#build",
+    "emoji-mart": "^2.2.1",
```

Comment:
https://github.com/missive/emoji-mart/pull/132 and https://github.com/missive/emoji-mart/pull/133 has been merged.